miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -987,6 +987,14 @@ Sample.hot = {
       }
     }
     
+    function validatePosReqdNumber (value, callback) {
+      if (Handsontable.helper.isNumeric(value) && value > 0) {
```

Comment:
This can also be:
`callback(Handsontable.helper.isNumeric(value) && value > 0);`